preppal-be/models/recipe.ts

```diff
@@ -50,6 +50,12 @@ const RecipeSchema = new Schema({
     },
     modifiedDate: {
         type: Date
+    },
+    tags: {
+        type: Array
```

Comment:
Is there a way to specify the type of the items in the Array as well?

Reply:
fixed.